### DISCUSSION TITLE
有人遇到过安卓手机可以访问而苹果手机无法访问的情况吗

### DISCUSSION THREAD

**Discussion by jiweiedu** [v2fly/v2ray-core#1963] [Q&A]
服务端：v2ray一键安装脚本
版本：5.1  管理脚本3.6
客户端：
安卓手机：v2rayNG
苹果手机：Shadowsocks

安卓可以用
苹果测速是有的，但是访问不通

**Comment by SwenXiao** [comment]
我也碰到了，有解决吗
